README.md

```diff
@@ -308,8 +308,8 @@ ort {
 }
 ```
 
-The _scanner_ creates a table called `scan_results` and stores the data in a
-[jsonb](https://www.postgresql.org/docs/current/datatype-json.html) column.
+While the specified schema already needs to exist, the _scanner_ will itself create a table called `scan_results` and
```

Comment:
Maybe a stupid question - where would the ORT user be able to get this schema?  Should we provide an example schema?

Reply:
This refers to the schema object in the PgSQL database. The required tables and indexes are automatically created by ORT already.

Reply:
AFAIU a "schema" is just a "container" for tables, and nothing with a given structure that you would need to provide.